doc/specs/vulkan/chapters/devsandqueues.txt

```diff
@@ -347,6 +347,13 @@ include::../structs/VkDeviceCreateInfo.txt[]
 
 include::../validity/structs/VkDeviceCreateInfo.txt[]
 
+fname:vkCreateDevice creates a VkDevice object, then enables
+extensions requested by the application. If an extension
+is provided by a layer, the layer must: be specified at
```

Comment:
The statement is true. "If an extension is provided by a layer, the layer **must** be specified at CreateInstance time."
We've moved that out of the "Valid Use" section as that implied the app needed to verify that the requested extension is supported/exists prior to calling Create(Instance|Device). That's not necessary. The intent of that statement is to make it clear that an application needs to do both parts - enable the layer and the extension to use the extension. Verses having the loader automatically enable layers based on extensions requested.